mmstudio/src/main/java/org/micromanager/display/internal/animate/AnimationController.java

```diff
@@ -89,9 +89,10 @@
 
    private final ScheduledExecutorService scheduler_ =
          Executors.newSingleThreadScheduledExecutor(ThreadFactoryFactory.
-               createThreadFactory("AnimationController"));
+               createThreadFactory("AnimationController", 4));
```

Comment:
Thread priority sounded nice to me, too, when I first saw it could be set, but the more you read about what it actually does (or doesn't) in each OS, the less attractive it seems.
See, e.g., https://vanillajava.blogspot.com/2011/09/why-thread-priority-rarely-matters.html.
I would vote for not introducing this, for simplicity's sake (a conscientious future developer might find it terrifying to modify this code if they wondered what behavior depends on the thread priority) -- since you say it doesn't make much of a difference.